pkg/cli/mirror/cincinnati_graph_image.go

```diff
@@ -87,13 +89,20 @@ func (o *MirrorOptions) buildGraphImage(ctx context.Context, dstDir string) (ima
 
 	// unpack graph data archive and build image
 	graphToFile := filepath.Join(dstDir, config.GraphDataDir, outputFile)
-	add, err := builder.LayerFromPath(".", graphToFile)
+	graphDataFolder := filepath.Join(dstDir, config.GraphDataDir, "/graph-data")
+
+	err = archiver.Unarchive(graphToFile, graphDataFolder)
```

Comment:
Before this change oc-mirror also handled tar.gz file, I am not sure why we need to unarchive the new tar.gz file?

Reply:
the old logic extracted the tar file when the container was started using Entrypoint. 
In cases when the entrypoint is not used, we wont be able to extract the contents of tar file. (for example using dkrv2 plugin of cincinnati). 
This change makes the container image compatible with both cases, i.e using as `init` container and using with dkrv2 plugin.